### PR TITLE
Update getting_started.adoc

### PR DESCRIPTION
**Pull request by jdmooney**
Edited Docker installation part

### CODE REVIEW

File: userguide/tutorials/getting_started.adoc [hunk 1]
```diff
@@ -1,40 +1,78 @@
 = Getting Started
 
+This document explains what you need to get started using KillBill in your chosen environment. The first step is installing KillBill itself and the Kaui administrative console, along with their required database, in the environment you will be using.
```
Comment (pierre) [inline userguide/tutorials/getting_started.adoc]: Nit: we usually spell `Kill Bill` (with a space).

**Review by pierre — APPROVED** [review]

File: userguide/tutorials/getting_started.adoc [hunk 2]
```diff
@@ -1,40 +1,78 @@
...
 
-Already familiar with AWS? Running Kill Bill on AWS using our http://docs.killbill.io/latest/aws.html[One-Click installer] is the easiest and fastest way to get started.
+=== AWS (One-Click)
+
+If you prefer to run KillBill on a cloud server, and are familiar with AWS, this is the easiest and fastest way to get started. Just try out our http://docs.killbill.io/latest/aws.html[One-Click installer]. 
 
 === Docker
 
-You can run Kill Bill locally using our official Docker images.
+You can run Kill Bill locally or in the cloud using our official Docker images. These require the installation of Docker and Docker Compose. Docker Compose is used to setup and manage the three separate Docker containers required by KillBill.
```
Comment (sbrossie) [inline userguide/tutorials/getting_started.adoc]: Maybe restate what those are. Something like:

, namely `killbill` server, `kaui` (admin UI) and the SQL engine

File: userguide/tutorials/getting_started.adoc [hunk 3]
```diff
@@ -71,27 +109,29 @@ services:
...
-and run:
+Now place this file in your current directory and run:
 
 [source,bash]
 ----
 docker-compose up
 ----
 
-3 containers will start:
+If all goes well 3 containers will start:
```
Comment (sbrossie) [inline userguide/tutorials/getting_started.adoc:119]: If you have any tips, based on what you have seen, on potential issues and how to track them, you could add some a `troubleshooting` section.

**Review by sbrossie — APPROVED** [review]
@jdmooney We left a few comments on small things you could change/improve, but overall we are ready to merge. Let us know when you had a chance to go through them, and then, we'll just merge

File: userguide/tutorials/getting_started.adoc (hunk 3 above)
Comment (jdmooney) [reply userguide/tutorials/getting_started.adoc:119]: I'm not sure if I know enough to do this yet, but I will work on it

File: userguide/tutorials/getting_started.adoc (hunk 2 above)
Comment (jdmooney) [reply userguide/tutorials/getting_started.adoc]: I added a statement for this

**Pull request merged**